pyvrp/GeneticAlgorithm.py

```diff
@@ -95,7 +95,7 @@ def __init__(
         # infeasible solution as the initial best.
         feas = [indiv for indiv in initial_solutions if indiv.is_feasible()]
         sols = feas if feas else initial_solutions
-        self._best = min(sols, key=self._cost_evaluator)
+        self._best = min(sols, key=self._cost_evaluator.penalized_cost)
```

Comment:
If we use `cost()` this whole filtering is not needed: just find the minimum in `initial_solutions`.